preview-src/index.ts

```diff
@@ -218,6 +221,16 @@ function addEventListeners(pr: PullRequest): void {
 			branch: pr.repositoryDefaultBranch
 		});
 	});
+
+	window.onscroll = () => {
```

Comment:
This event fires really frequently, so I think we should throttle posting messages.

Maybe something like using `setTimeout` for changes to `pendingCommentText` above

Reply:
Good thought! I hope it's okay that I've added `debounce` as a dependency for this purpose.